src/helpers/validators/validateEmail.ts

```diff
@@ -0,0 +1,42 @@
+// Import helpers
+import validateRegex from './shared/helpers/validateRegex';
+
+// Import types
+import ValidationResult from './shared/types/ValidationResult';
+
+/**
+ * Determines whether a given email address is valid.
+ * @author Austen Money
+ * @param email email address to validate
+ * @returns whether email fulfills proper formatting requirements, includes a
+ *   cleaned version of the address without leading or trailing
+ *   whitespace if valid or an error message if invalid.
+ */
+const validateEmail = (email: string): ValidationResult => {
+  // validation regex, sourced from HTML living standard
```

Comment:
Do you have a link for this source?

Reply:
Just added! Is there a specified format for incorporating links into comments?

Reply:
Unfortunately not :/ this doesn't happen too often so it's awkward every time. What you wrote is fine.